workspace/python/ds-pack/v1/requirements.txt

```diff
@@ -0,0 +1,27 @@
+boto3==1.23.7
+ipywidgets==7.6.5
+jupyter-lsp==1.5.1
+jupyter-server-proxy==3.2.1
+jupyterhub==1.4.2
+jupyterlab-git==0.32.4
+jupyterlab-lsp==3.10.0
+jupyterlab==3.1.11
+jupyterlab_widgets==1.1.0
+jupytext==1.13.8
+lckr-jupyterlab-variableinspector==3.0.9
+mypy-ls==0.5.1
+nbconvert==6.2.0
+notebook==6.4.3
+pandas==1.4.2
+psycopg2==2.9.3
+pyls-black==0.4.7
+pyls-flake8==0.4.0
+pyls-isort==0.2.2
+pyls-mypy==0.1.8
+python-lsp-black==1.0.0
+python-lsp-server[all]==1.4.1
+scikit-learn==1.1.1
+snowflake-connector-python[pandas]==2.7.7
+snowflake==0.0.3
```

Comment:
This is a unknown [package](https://pypi.org/project/snowflake/) - potentially security risk